app/ide-desktop/lib/client/src/index.ts

```diff
@@ -5,23 +5,33 @@
  * Inter-Process Communication channel, which enables seamless communication between the served web
  * application and the Electron process. */
 
+import * as fs from 'node:fs/promises'
+import * as fsSync from 'node:fs'
+import * as pathModule from 'node:path'
+import * as process from 'node:process'
+
+import * as electron from 'electron'
+
+import * as content from 'enso-content-config'
+
 import * as authentication from 'authentication'
 import * as config from 'config'
 import * as configParser from 'config/parser'
-import * as content from 'enso-content-config'
 import * as debug from 'debug'
-import * as electron from 'electron'
-import * as fs from 'node:fs/promises'
-import * as fss from 'node:fs'
 import * as ipc from 'ipc'
 import * as naming from 'naming'
-import * as path from 'node:path'
 import * as paths from 'paths'
 import * as projectManager from 'bin/project-manager'
 import * as security from 'security'
 import * as server from 'bin/server'
-import * as process from 'process'
-const logger = content.logger
+
+// =================
+// === Constants ===
+// =================
+
+const LOGGER = content.LOGGER
```

Comment:
This is not a constant in the "traditional" meaning. This is just a variable pointing to a class instance, so IMO this should be written `logger`, just like `console` is `console` and not `CONSOLE`.